src/ert/gui/simulation/experiment_panel.py

```diff
@@ -297,3 +313,43 @@ def validationStatusChanged(self) -> None:
             self.run_button.text() == EXPERIMENT_READY_TO_RUN_BUTTON_MESSAGE
             and widget.isConfigurationValid()
         )
+
+    def populate_clipboard_debug_info(self) -> None:
+        output = ""
+        kv = {"**Platform**": "", ":-----": ":-----"}
+        kv["Date"] = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+        kv["OS"] = (
+            platform.system() + " " + platform.release() + " " + platform.machine()
+        )
+        kv["Hostname"] = platform.node()
+
+        for e in ["KOMODO_RELEASE"]:
+            if e in os.environ:
+                kv[e.capitalize()] = os.environ[e]
+
+        kv["Python Version"] = platform.python_version()
+
+        kv["**Ensemble**"] = ""
+        kv["Queue"] = self.config.queue_config.queue_system.name
+        kv["Simulation Mode"] = self.get_current_experiment_type().name()
+        kv["Config file"] = self._config_file
+        kv["Ensemble Path"] = self.config.ens_path
+        kv["Ensemble Size"] = str(self._model.ensemble_size)
+        kv["Max Running"] = str(self.config.queue_config.queue_options.max_running)
+
+        kv["**Status**"] = ""
+        kv["Running time"] = (
+            format_running_time(self._model.get_runtime()).split(":")[1].strip()
+        )
+
+        for status, count in self._model.get_current_status().items():
+            kv[status] = str(count)
+
+        kv["Max Memory Consumption"] = byte_with_unit(
```

Comment:
memory for the ert process or max of a realization (we have both, and should be precise)

Reply:
This would be the max for all forward model steps.
Let's find the other one too! 👍